src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -99,6 +100,7 @@ object SamResourceActions {
 
 object SamResourceTypes {
   val resourceTypeAdminName = ResourceTypeName("resource_type_admin")
+  val googleProject = ResourceTypeName("google-project")
```

Comment:
Other resource types seem to be named in snake case. Should this one too to be consistent?